templates/home/templates/product-is-template.html.twig

```diff
@@ -8,7 +8,15 @@
                     <img class="mx-auto d-block pb-2 product-is-icon" src="{{ asset(icon_path) }}"
                          alt="{{ item.icon_alt|trans }}">
                     <h5 class="py-3">{{ item.title |trans }}</h5>
-                    <p class="font-size-1-1">{{ item.text|trans }}</p>
+                    <p class="font-size-1-1">
+                        {% if item.text == 'rp_is_satisfying_p' %}
+                            {{ item.text|trans({'%link%': 'https://reconnect.fr/annexe/formation-rp'})|sanitize_html|raw }}
+                        {% elseif item.text == 'cfn_vault_is_satisfying_p' %}
+                            {{ item.text|trans({'%link%': 'https://reconnect.fr/annexe/formation-cfn'})|sanitize_html|raw }}
+                        {% else %}
+                            {{ item.text|trans }}
+                        {% endif %}
+                    </p>
```

Comment:
Instead of using conditionnal rendering using hard coded strings, you could add a property to particular items

Reply:
You can use path function here too instead of hard coded urls